scripts/lib/CIME/case_run.py

```diff
@@ -120,7 +121,7 @@ def _run_model_impl(case, lid):
                         restore_from_archive(case)
 
                         orig_cont = case.get_value("CONTINUE_RUN")
-                        if not orig_cont:
+                        if not (orig_cont or skip_pnl):
```

Comment:
You still want to change the CONTINUE_RUN value in either case, so the skip_pnl should be a seperate conditional
if not orig_cont:
   case.set_value("CONTINUE_RUN", True)
   if not skp_pnl:
       create_namelists(case)